sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -219,37 +251,28 @@ Specifies whether the data symbols in the pipe are in big-endian
 order.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*
+or *avalon_mm*.
 
 |`protocol`
 | Specifies the protocol for the pipe interface. Currently, the protocols supported
-are: *avalon_streaming*, *avalon_streaming_uses_ready*, *avalon_mm*, and *avalon_mm_uses_ready*.
+are: *avalon_streaming*, *avalon_mm*, and *axi_streaming*.
 
 *avalon_streaming*
 
 Provide an Avalon streaming interface as described in https://www.intel.com/content/www/us/en/docs/programmable/683091/22-3/introduction-to-the-interface-specifications.html[Intel® Avalon Interface Specifications].
 
-With this choice of protocol, no ready signal is exposed by the host pipe, and the sink cannot backpressure.
-
-*avalon_streaming_uses_ready*
-
-Provide an Avalon streaming interface as described in https://www.intel.com/content/www/us/en/docs/programmable/683091/22-3/introduction-to-the-interface-specifications.html[Intel® Avalon Interface Specifications].
-
-This protocol allows the sink to backpressure by deasserting the ready signal asserted. The sink signifies that it is ready to consume data by asserting the ready signal. 
-
 *avalon_mm*
 
 Provide an Avalon memory mapped interface as described in https://www.intel.com/content/www/us/en/docs/programmable/683091/22-3/introduction-to-the-interface-specifications.html[Intel® Avalon Interface Specifications].
```

Comment:
This doesn't mention anything about the interface being shared.

Reply:
Does it make sense to write this as a requirement ("this interface will be shared") or as a possibility ("this interface may be shared")?

Reply:
On second thought I guess we can leave it as is.  Saying nothing here gives us the maximum flexibility in the future with regards to the default behaviour if we ever decide to add another control for this.